sklearn/datasets/_base.py

```diff
@@ -1193,13 +1194,16 @@ def load_linnerud(*, return_X_y=False, as_frame=False):
     data_filename = "linnerud_exercise.csv"
     target_filename = "linnerud_physiological.csv"
 
+    data_module_path = resources.files(DATA_MODULE)
     # Read header and data
-    with _open_text(DATA_MODULE, data_filename) as f:
+    data_path = data_module_path / data_filename
+    with data_path.open("r") as f:
```

Comment:
Here the dataset file is fixed, we should hardcode the matching encoding in the code.

```suggestion
    with data_path.open("r", encoding="utf-8") as f:
```